EFCore.CheckConstraints/Internal/ValidationCheckConstraintConvention.cs

```diff
@@ -22,6 +22,14 @@ public class ValidationCheckConstraintConvention : IModelFinalizingConvention
 
     public const string DefaultUrlAddressRegex = @"^(http://|https://|ftp://)";
 
+    public const string SqlServerDatabaseProviderName = "MICROSOFT.ENTITYFRAMEWORKCORE.SQLSERVER";
+
```

Comment:
nit: remove empty lines between the different consts

Reply:
I have removed the empty lines and removed the casting to uppercase before comparison.
Kept the centralization of names.

Reply:
I committed to my main branch, i'm not an expert on github, do i need to do other steps to add the last commit to this pull request?